File: UserProfilerModel/ListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UserProfilerModel
{
    public class ListModel
    {

        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Zip { get; set; }

        public string Title { get; set; }
        public string Description { get; set; }
        public int? NoOfBedrooms { get; set; }
        public int? NoofBeds { get; set; }
        public int? NoOfPeople { get; set; }

        public int? NightlyCost { get; set; }
        public int? WeeklyCost { get; set; }

        public bool IsPetsAllowed { get; set; }
        public bool IsWifiAvailable { get; set; }
        public bool IsACAvailable { get; set; }
        public bool IsLiftAvailable { get; set; }
        public bool IsPrivatePool { get; set; }
        public bool IsBuzzer { get; set; }


    
    }
}

File: UserProfilerWeb/Scripts/Listing/ListModel.js

function ListModel() {

    this.address1 = ko.observable();
    this.address2 = ko.observable();
    this.city = ko.observable();
    this.state = ko.observable("");
    this.zip = ko.observable("");
    this.country = ko.observable();

    this.title = ko.observable();
    this.description = ko.observable();
    this.noofbedrooms = ko.observable();
    this.noofbeds = ko.observable("");
    this.noofpeople = ko.observable("");
    this.propertyType = ko.observable();


    this.nightlyCost = ko.observable();
    this.weeklyCost = ko.observable();


    this.isPetsAllowed = ko.observable();
    this.isWifiAvailable = ko.observable();
    this.isACAvailable = ko.observable();
    this.isLiftAvailable = ko.observable();
    this.isPrivatePool = ko.observable();
    this.isBuzzer = ko.observable();
    

}
